tests/integration/logging.tcl

```diff
@@ -28,10 +28,14 @@ if {$backtrace_supported} {
             if {$threads_mngr_supported} {
                 assert_equal [count_log_message 0 "failed to open /proc/"] 0
                 assert_equal [count_log_message 0 "failed to find SigBlk or/and SigIgn"] 0
-                verify_log_message 0 "*redis-server *\(aeMain*" 0
+                assert_equal [count_log_message 0 "threads_mngr: waiting for threads' output was interrupted by signal"] 0
+                assert_equal [count_log_message 0 "threads_mngr: waiting for threads' output timed out"] 0
```

Comment:
After the `r debug sleep 1` returns, we know we have a log file, no?

Reply:
no.. maybe the system is super slow, and redis is starved of CPU time.
the code below uses wait_for to wait for the watchdog to kick in and print.
so all the other validations should be after

Reply:
Ok, but if we get one of these error messages, we will fail to find the stacktraces and not for having the errors, and the test doesn't indicate the actual problem.

I changed the test to get a full report by failing the server and after we checked we got to the end of the log report, we can start checking that the backtraces prints succeeded.
```
catch {r debug segfault}
wait_for_log_messages 0 {"*REDIS BUG REPORT END*"} 0 100 100
```

I could replace 
`assert_equal [count_log_message 0 "failed to open /proc/"] 0` with
`catch {wait_for_log_messages 0 {"failed to open /proc/"} 0 100 100}` 
but in the typical case, where we fail and retry to find a line that doesn't exist because there was no error, the test is very slow.